jenkins/common.groovy

```diff
@@ -93,6 +93,13 @@ def puppeteerNotification() {
     failOnError: true
 }
 
+def slackIntegrationNotify() {
+  message = "(Testing) @jbalboni: integration tests failed. |${env.RUN_DISPLAY_URL}".stripMargin()
```

Comment:
I plan on iterating on this. Ideally, we'd send a message to a slack channel that isn't the deploys one.